Tests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+public class Tests
+{
+    static void ExpectTrue(bool expression)
+    {
+        if (!expression)
+        {
+            Console.WriteLine("Expected true, but got false");
+            Environment.Exit(1);
+        }
+    }
+    static void ExpectFalse(bool expression)
+    {
+        if (expression)
+        {
+            Console.WriteLine("Expected false, but got true");
+            Environment.Exit(1);
+        }
+    }
+
+    public static void PerformTests()
+    {
+        //test valid values
+        CompositeBatteryParameter compositeParam = new CompositeBatteryParameter(25, 70, 0.7f);
+        ExpectTrue(BatteryChecker.batteryIsOk(compositeParam, Console.WriteLine));
+
+        compositeParam = new CompositeBatteryParameter(0, 20, 0.0f);
+        ExpectTrue(BatteryChecker.batteryIsOk(compositeParam, Console.WriteLine));
+
+        compositeParam = new CompositeBatteryParameter(45, 80, 0.8f);
+        ExpectTrue(BatteryChecker.batteryIsOk(compositeParam, Console.WriteLine));

```

Comment:
Test cases to verify language strings need to be present. Right now there is no way to verify if the right strings have been printed!